fe/src/main/java/org/apache/doris/analysis/ShowPluginsStmt.java

```diff
@@ -27,10 +27,13 @@
     private static final ShowResultSetMetaData META_DATA =
             ShowResultSetMetaData.builder()
                     .addColumn(new Column("Name", ScalarType.createVarchar(64)))
-                    .addColumn(new Column("Status", ScalarType.createVarchar(10)))
-                    .addColumn(new Column("Type", ScalarType.createVarchar(80)))
-                    .addColumn(new Column("Library", ScalarType.createVarchar(64)))
-                    .addColumn(new Column("License", ScalarType.createVarchar(80)))
+                    .addColumn(new Column("Type", ScalarType.createVarchar(10)))
+                    .addColumn(new Column("Description", ScalarType.createVarchar(200)))
+                    .addColumn(new Column("Version", ScalarType.createVarchar(20)))
+                    .addColumn(new Column("JavaVersion", ScalarType.createVarchar(20)))
+                    .addColumn(new Column("ClassName", ScalarType.createVarchar(64)))
+                    .addColumn(new Column("SoName", ScalarType.createVarchar(64)))
+                    .addColumn(new Column("Sources", ScalarType.createVarchar(200)))
```

Comment:
How about adding a `Status` column to show that if the plugin is loaded successfully.